dev-java/jetty-util/jetty-util-9.3.5.20151012.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+JAVA_PKG_USE="doc source"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Jetty's Utils"
+
+MY_PN="jetty"
+MY_PV="${PV/2015/v2015}"
+MY_P="${MY_PN}-${MY_PV}"
+
+SLOT="$(get_version_component_range 1-2)"
```

Comment:
Two wrongs don't make a right here. I estimate that a whole 3 ebuilds are affected.

```
% find -name "*.ebuild" -exec sh -c 'fgrep -q java-pkg {} && fgrep -q _version_ {} && ( fgrep -q versionator {} || echo {} )' \;
./dev-java/poi/poi-3.0.2-r1.ebuild
./dev-java/htmlparser-org/htmlparser-org-1.6.ebuild
./dev-java/vecmath/vecmath-1.6.0_pre12.ebuild
```